packages/taxdat/R/plot_DroppedData_table.R

```diff
@@ -30,7 +30,8 @@ plot_DroppedData_table <- function(config, cache, cholera_directory, aesthetic =
                                 u_OCs  = paste(sort(unique(OC_UID)), collapse = ",")
   )
   
-  used_obs_stats <- obs_stats[order(obs_stats$year), ]
+  used_obs_stats <- obs_stats[order(obs_stats$year), ] %>% 
+    dplyr::filter(!grepl('imputed',u_OCs))

```

Comment:
Are we filtering the imputed obs because imputed obs are not included in the original obs_stats object?

Reply:
yes, imputed observations are not included in the sf_cases but are included in the sf_cases_resized, so if we include them, there will be cases that when we subtract the number of cases/observations between sf_cases and sf_cases_resized, there will be errors.